test/test_public_bindings.py

```diff
@@ -271,7 +271,7 @@ def test_no_new_bindings(self):
         self.assertTrue(torch_C_bindings.issubset(torch_C_allowlist_superset), msg)
 
     # AttributeError: module 'torch.distributed' has no attribute '_shard'
-    @unittest.skipIf(IS_WINDOWS, "Distributed Attribute Error")
+    @unittest.skipIf(IS_WINDOWS or IS_JETSON, "Distributed Attribute Error")
```

Comment:
This feels like a config problem to me (i.e. you compile pytorch without some deps, but whatever)

Reply:
yes pytorch is built w/o distributed for jetsons